test/export/test_hop.py

```diff
@@ -84,6 +84,11 @@ def forward(self, *args):
             kwargs = inp.kwargs
             ep = export(model, args, kwargs)
             self._compare(model, ep, args, kwargs)
+        # Mannually resetting dynamo state to avoid memory leak warning.
+        # Note that even though TestCase.tearDown calls reset, the
+        # memory leak checks when PYTORCH_TEST_CUDA_MEM_LEAK_CHECK=1
+        # checks memory leaks before that.
+        torch._dynamo.reset()
```

Comment:
Thanks for the quick diagnosis!

It sounds like there is no real memory leak since we would have called _dynamo.reset() later, just that the check happens before we can reset()?

Reply:
yeah, that's the problem.